screen/accounts/src/main/java/com/ivy/accounts/compute/DataSource.kt

```diff
@@ -0,0 +1,8 @@
+package com.ivy.accounts.compute
+
+import com.ivy.data.model.Transaction
+
+sealed interface DataSource {
+    data object DB : DataSource
```

Comment:
Just make it accept List<Transaction> this seems to open the room for confusion and errors